docs/quickstart.rst

```diff
@@ -13,7 +13,7 @@ create a minimal application, or use its scaffolding tool:
 
 ::
 
-    pcreate -s starter MyProject
+    $ pcreate -s starter MyProject
```

Comment:
We could add this information visually maybe? Oh well, that's not that important; we can keep it like that :)